test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -136,6 +136,7 @@ UcxContext::~UcxContext()
     }
 }
 
+ucp_request_param_t UcxContext::param;
```

Comment:
1. pls rename it to recv_param
2. space line after
3. move it to just before line 90

Reply:
@yosefe Done. Waiting for CI result.